kong/runloop/plugins_iterator.lua

```diff
@@ -92,6 +92,10 @@ local function get_next(self)
 
   self.i = i
 
+  if not self.configured_plugins[plugin.name] then
```

Comment:
Wondering, can be somehow join `loaded_plugins` and `configured_plugins` so that we don't needlessly loop them?

Reply:
i had initially considered that, but doing so makes the code very difficult to re-use. re-executing the loop here during initialization is worth it IMO to keep things sane

Reply:
Fair.